apps/emacs/emacs.talon

```diff
@@ -14,7 +14,7 @@ execute <user.text>$:
     user.emacs_meta_x()
```

Comment:
There's still a few uses of `emacs_meta_x` in emacs.talon that need updating.

Reply:
Ok, should have fixed this now.